CMakeLists.txt

```diff
@@ -607,17 +607,20 @@ if(EL_TESTS)
       set_source_files_properties("${DRIVER}" PROPERTIES
         OBJECT_DEPENDS "${PREPARED_HEADERS}")
       target_link_libraries(tests-${TYPE}-${TESTNAME} El)
+      if(NOT NO_BINARY_SUBDIRECTORIES)
```

Comment:
It is generally considered good form to avoid double-negatives if it doesn't change the control flow too much. For example, if you switched the `if` and `else` branches, it goes away.